.github/workflows/documentation-windows.yml

```diff
@@ -2,7 +2,7 @@ name: Documentation Windows
 
 on:
   push:
-    branches: [ master, develop ]
+    branches: [ master, develop, debug_windows_doc ]
```

Comment:
I don't think we need this, right?

Reply:
not really in general no. I kinda like the idea of just being able to push to that branch without having to mod the workflow file, but this can go yes.